### DISCUSSION TITLE
Selecting Specific Layers from Illustrator Files for Conversion

### DISCUSSION THREAD

**Discussion by trythisone35** [ImageMagick/ImageMagick#6625] [Help]
### ImageMagick version

7.1.1-15-Q16-HDRI-x64

### Operating system, version and so on

Windows 10

### Description

I have Adobe Illustrator files that have multiple layers.  One layer has annotations on it.  Is it possible to not include a specific layer or artboard as part of the conversion to jpg?

I attached a simple example with a red blob on one layer and the rest of the artwork on the other.  I would want to convert the image without the presence of the red blob.

Any suggestions on how to do this would be greatly appreciated.

(Note: I have installed ghostscript for the conversion of the ai file) 
[mylayer.zip](https://github.com/ImageMagick/ImageMagick/files/12542933/mylayer.zip)
![test1](https://github.com/ImageMagick/ImageMagick/assets/79946966/82457a54-1340-46e7-aa4d-99792ee9f704)

**Comment by fmw42** [comment]
You have uploaded a PNG.  PNG does not support layers.  Please post your illustrator file.

**Comment by snibgo** [comment]
The AI is in the posted zip file.

IM passes the entire AI file to Ghostscript, which rasterizes it and returns one image to IM. IM can't see different layers within the AI file.

Perhaps Ghostscript has command-line options to do what you want.

**Reply by fmw42** [reply]
Sorry, I overlooked the zip file link.

**Reply by trythisone35** [reply]
No worries.  Thanks for the response.  Is there a similar forum/community for Ghostscript?  I couldn't find the quality and quantity of information like this one.
